blocks/procedures.js

```diff
@@ -946,68 +967,68 @@ Blockly.Blocks['procedures_callnoreturn'] = {
          *   </block>
          * </xml>
          */
-        const xml = Blockly.utils.xml.createElement('xml');
-        const block = Blockly.utils.xml.createElement('block');
+        const xml = xmlUtils.createElement('xml');
+        const block = xmlUtils.createElement('block');
         block.setAttribute('type', this.defType_);
         const xy = this.getRelativeToSurfaceXY();
-        const x = xy.x + Blockly.SNAP_RADIUS * (this.RTL ? -1 : 1);
-        const y = xy.y + Blockly.SNAP_RADIUS * 2;
+        const x = xy.x + internalConstants.SNAP_RADIUS * (this.RTL ? -1 : 1);
```

Comment:
Flagging a use of `internalConstants` for #5158. Internals are leaking here.

Reply:
Noted, but as discussed will leave this as is for now.